pb/yara.proto

```diff
@@ -204,22 +213,30 @@ message IntegerFunction {
   optional Expression argument = 2;
 }
 
-// Expression for iterating over string occurrences.
-// Example: for any i in (1..10) : @s1[i] != @s2[i]
+// Expression for iterating over iterators.
+// Example: for <for_expression> k,v in iterator : ( expression )
+// Next id: 6
 message ForInExpression {
   // FOR expression: "for any". Required.
   optional ForExpression for_expression = 1;
 
-  // Identifier to use in the expression to match: "i". Required.
-  optional string identifier = 2;
+  // Identifiers that will hold the valies returned by the iterator. Required.
+  repeated string identifiers = 2;
 
-  // Integer set: "(1..10)". Required.
-  optional IntegerSet integer_set = 3;
```

Comment:
It's safer to rename a field instead of deleting it because, otherwise, we may use the same field number in the future accidentally (https://developers.google.com/protocol-buffers/docs/proto#updating).

Reply:
Yes, that's important for a proto that has been stored somewhere, or that is being used for exchanging information between two systems that are not updated at the same time. But in this case I think it's less confusing if we don't leave deprecated fields around.